### PR TITLE
Add GnuPG maintained packages

### PR DESCRIPTION
**Pull request by andreasstieger**
update gpa
add gnupg, pinentry, npth, libassuan, libgcrypt, libgpg-error, libksba, gpgme

### CODE REVIEW

**Comment by vuntz** [comment]
Thanks for the patch!

It's kind of suboptimal, but you also need to add the packages to upstream-packages-match.txt even when the names match (just add `libassuan:` to the file, for instance). Otherwise, the versions won't be taken into account.

**Comment by andreasstieger** [comment]
Oh didn't know. d192e02 should fix that.

File: server/upstream/upstream-packages-match.txt [hunk 1]
```diff
@@ -689,6 +694,7 @@ libical-glib:
 libinfinity:
 libiptcdata:
 libjingle:
+libksba
```
Comment (vuntz) [inline server/upstream/upstream-packages-match.txt]: Missing `:` at the end.

**Comment by andreasstieger** [comment]
f4a4d07 adds the missing <code>:</code>

**Comment by vuntz** [comment]
Thanks; this will be part of the next run (within the next 12 hours).

**Pull request merged**